inc/head.php

```diff
@@ -2,18 +2,33 @@
   session_start();
   require_once('servervars.php');
 
+  $kmw_tiers = array('alpha', 'beta', 'stable');
+  $kmw_builds = array();
+
+  if(isset($_REQUEST['version']) && preg_match('/^(\d+)\.(\d+)\.(\d+)$/', $_REQUEST['version'])) {
+    $kmwbuild = $_REQUEST['version'];
+  } else {
+    foreach ($kmw_tiers as $index => $possible_tier) {
+      $kmw_builds[$possible_tier] = get_keymanweb_version($possible_tier);
```

Comment:
This means 3 requests to api.keyman.com for each page load. I don't think we should do this!

One possible way around this is to rework this to use our internal downloads.keyman.com version api, which is what api.keyman.com reads from anyway. If we do go this way, we should comment accordingly that the downloads.keyman.com api is internal and subject to change, in case other people decide to try and use the same pattern.

https://downloads.keyman.com/api/version/web returns a perfect result for your needs:

```json
{
    "web": {
        "alpha": "15.0.18",
        "beta": "14.0.248",
        "stable": "13.0.111"
    }
}
```